modules/standard/BigInteger.chpl

```diff
@@ -2212,6 +2212,18 @@ module BigInteger {
 
 
   // divexact
+
+/*
+
+Computes the quotients when corresponding elements of a bigint array *n* are divided by corresponding elements of a bigint array *d*, and stores the quotient in another bigint array preceding the function call for whichever elements of the array are exactly divisible, with remainder zero.
```

Comment:
Generally speaking, we keep Chapel module source code to 80 columns (in other words you should add newlines to this to make the lines shorter).

Reply:
The GMP manual for this function is here: https://gmplib.org/manual/Integer-Division.html

It describes the function in this way:
> Set q to n/d. These functions produce correct results only when it is known in advance that d divides n.
> 
> These routines are much faster than the other division functions, and are the best choice when exact division is known to occur, for example reducing a rational to lowest terms. 

So there are a differences between the new documentation above and the description from GMP:
 * shorter math-y formulation `n/d`
 * the term "bigint array"
 * less emphasis on the function producing incorrect results if n is not divisible by d

I think it'd be better to be closer to the GMP documentation in these regards.

Reply:
And to mention that it's expected to be faster than regular division (where it is not assumed that d divides n)